Lombiq.DataTables/Services/IndexBasedDataTableDataProvider.cs

```diff
@@ -34,8 +34,9 @@ protected IndexBasedDataTableDataProvider(IDataTableDataProviderServices service
     public override async Task<DataTableDataResponse> GetRowsAsync(DataTableDataRequest request)
     {
         var query = new SqlBuilder(_session.Store.Configuration.TablePrefix, _session.Store.Configuration.SqlDialect);
+        var schema = _session.Store.Configuration.Schema;
         query.Select();
-        query.Table(typeof(TIndex).Name);
+        query.Table(typeof(TIndex).Name, alias: null, schema);
```

Comment:
The variable above is not necessary.

```suggestion
        query. Table(typeof(TIndex).Name, alias: null, _session.Store.Configuration.Schema);
```